src/index.js

```diff
@@ -57,7 +68,8 @@ submitButton.addEventListener("click", () => {
     const splitItems = splitText(text);
 
     if (splitItems) {
-        createListItems(splitItems);
+        const { fontColor, backgroundColor } = getConfiguration();
+        createListItems(splitItems, { fontColor: fontColor, backgroundColor: backgroundColor });
```

Comment:
```suggestion
        createListItems(splitItems, { fontColor, backgroundColor });
```

Aqui vc pode usar shorthand notation: https://medium.com/swlh/es6-object-property-shorthand-make-those-objects-look-cleaner-9392c6ebb18d